boefjes/boefjes/dependencies/plugins.py

```diff
@@ -150,12 +150,7 @@ def _put_normalizer(self, normalizer_id: str) -> None:
     def delete_settings(self, organisation_id: str, plugin_id: str):
         self.config_storage.delete(organisation_id, plugin_id)
 
-        try:
-            self._assert_settings_match_schema({}, organisation_id, plugin_id)
-        except SettingsNotConformingToSchema:
-            logger.warning("Making sure %s is disabled for %s because settings are deleted", plugin_id, organisation_id)
-
-            self.set_enabled_by_id(plugin_id, organisation_id, False)
+        # We don't check the schema anymore because we can provide entries through the global environment as well
```

Comment:
I think It would  be better if we 'optionally' skip this validation. That way we can do validation by default, but offer the user to skip validation if they want to do so by re-posting the form, with an extra argument.

Reply:
Shouldn't we just handle this in Rocky instead of requiring a call towards the KATalogus while we already know that the schema doesn't match?

Reply:
Oh, yes. I did not notice this was the kat-alogus.